acceptance/tests/resource/service/should_query_all.rb

```diff
@@ -0,0 +1,11 @@
+test_name "should query all services"
+
+agents.each do |agent|
+  next if agent == master
+
+  step "query with puppet"
+  on(agent, puppet_resource('service'), :accept_all_exit_codes => true) do
+    assert_equal(exit_code, 0, "'puppet resource service' should have an exit code of 0")
+    assert_match(stdout, /^service/, "'puppet resource service' should present service details")
```

Comment:
If puppet issues some other output prior to the service listing, ie. a deprecation of some kind, will this still match?